kube-janitor/values.yaml

```diff
@@ -14,10 +14,10 @@ kubejanitor:
 cron:
   schedule: "*/1 * * * *"
 
-resources: {}
```

Comment:
This is actually a common practice with helm charts.

> We usually recommend not to specify default resources and to leave this as a conscious choice for the user. This also increases chances charts run on environments with little resources, such as Minikube. If you do want to specify resources, uncomment the following lines, adjust them as necessary, and remove the curly braces after 'resources:'.

Usually it's intended to be set at deploy time.

Other examples...
https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L140
https://github.com/helm/charts/blob/master/stable/fluentd/values.yaml#L114

Reply:
Okay with me. I have seen among best practices assigning reasonable requests/limits and allowing chart users to override. Still, no issue really. I'll keep watching the chart community for any growing consensus.